app/templates/errors/500.html

```diff
@@ -1,5 +1,20 @@
 {% extends "_base_page.html" %}
 
+{% block page_title %}Internal server error - 500 - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+      with items = [
+          {
+              "link": url_for('main.index'),
+              "label": "Admin home"
+          }
+      ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
```

Comment:
We repeat this default breadcrumb stuff in a lot of places in the admin - is it possible to put it in the `_base_page.html` block? Or is visibility per page preferable?

Reply:
But then the index page would get a breadcrumb to itself, which we probably don't want?

Could override it there with an empty block I guess?  

I don't mind having each template responsible for its own breadcrumb - it just so happens that in admin most things are one page away from the home page, but I don't think that necessarily means we should make it generic.

Reply:
Fair enough. If we did decide to change it, it should probably go in a separate PR anyway.